### PR TITLE
update: move magicLinkForm disclaimer above confirm button

### PR DESCRIPTION
**Pull request by nickfranciosi**
update legal copy
move copy above button for magic link form
fixes https://github.com/lonelyplanet/dotcom-profile/issues/143

### CODE REVIEW

File: src/components/multiStep/multiStepLogin.jsx [hunk 1]
```diff
@@ -38,12 +45,12 @@ const MultiStepLogin = ({
...
 
   const EmailDisclaimer = (
-    <AuthDisclaimer>
-      By clicking next, you agree to our <a href="//www.lonelyplanet.com/legal/website-terms/">terms of service </a>
-       and that you have read our <a href="//www.lonelyplanet.com/legal/privacy-policy/">privacy policy</a>, including
-       our <a href="//www.lonelyplanet.com/legal/cookies/">cookie use</a>.
-       For additional account inquiries
-       see <a href="http://support.lonelyplanet.com/hc/en-us/sections/203968787-Account-administration" target="_blank" rel="noopener noreferrer">Account help</a>.
+    <AuthDisclaimer style={styles.disclaimerAboveButton}>
+      By clicking next below and creating an account, you agree to our <a href="//www.lonelyplanet.com/legal/website-terms/">terms of service </a>
```
Comment (thomasthesecond) [inline src/components/multiStep/multiStepLogin.jsx]: Can we put the space on the outside of the `</a>`? And should we use https on the `href`?

File: src/components/multiStep/multiStepLogin.jsx [hunk 2]
```diff
@@ -38,12 +45,12 @@ const MultiStepLogin = ({
...
+       service and that you're happy for Lonely Planet to use your information as set out in our <a href="//www.lonelyplanet.com/legal/privacy-policy/">privacy policy</a> (including our
+       <a href="//www.lonelyplanet.com/legal/cookies/"> cookie use</a>).
```
Comment (thomasthesecond) [inline src/components/multiStep/multiStepLogin.jsx]: Also here, space outside of `</a>` and https on lines 50 and 51

**Comment by nickfranciosi** [comment]
@thomasthesecond updated

File: src/components/multiStep/multiStepLogin.jsx [hunk 3]
```diff
@@ -30,20 +37,19 @@ const MultiStepLogin = ({
           event.preventDefault();
         }}
       >username and password</a> Lonely Planet uses cookies to improve your
-      experience, see our <a href="//www.lonelyplanet.com/legal/cookies/">Cookie Policy</a>.
+      experience, see our <a href="https//www.lonelyplanet.com/legal/cookies/">Cookie Policy</a>.
       You may receive notifications from us and can opt out at any time.
       For additional account inquiries
-      see <a href="http://support.lonelyplanet.com/hc/en-us/sections/203968787-Account-administration" target="_blank" rel="noopener noreferrer">Account help</a>.
+      see <a href="https://support.lonelyplanet.com/hc/en-us/sections/203968787-Account-administration" target="_blank" rel="noopener noreferrer">Account help</a>.
```
Comment (thomasthesecond) [inline src/components/multiStep/multiStepLogin.jsx]: You're going to hate me, sorry, I just double checked this and https doesn't work with support.lonelyplanet.com. I thought all our links use https now. My guess is we don't host this.

**Pull request merged**